Add missing include statement for compatibility with libstdc++ 11.

Taken from upstream:

  https://github.com/llvm/llvm-project/commit/b498303066a63a203d24f739b2d2e0e56dca70d1

diff --git a/utils/benchmark/src/benchmark_register.h b/utils/benchmark/src/benchmark_register.h
--- a/utils/benchmark/src/benchmark_register.h
+++ b/utils/benchmark/src/benchmark_register.h
@@ -1,6 +1,7 @@
 #ifndef BENCHMARK_REGISTER_H
 #define BENCHMARK_REGISTER_H
 
+#include <limits>
 #include <vector>
 
 #include "check.h"
